### DISCUSSION TITLE
Stylistic question: Multiple requests in one form

### DISCUSSION THREAD

**Discussion by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#3472] [Q&A] [answered]
Hi,

To the Devs: How would you prefer that we handled multiple requests, sometimes of different types? This something of a two part question: First of all, what's the best way to request a number of related copies of systems, such as SA-7, SA-14 and SA-16 teams for a country that operates all three? Create three issues, or just one and explain in the comments field? Secondly, are the large 'overhaul' posts that I've been making (#2581 for example) useful or should I be attempting to separate out the issues into a couple of dozen individual tasks? I have been assuming that a single post is the best way to show the rationale behind the changes and that all the edits will be done by one person in a single sitting, but that might not be your workflow.

**Comment by PygmalionOfCyprus** [comment, marked as answer]
There isn't really a "correct" answer for this; we've seen both and we can handle either. A lot of it comes down to your workflow. For the SA-7/SA-14/SA-16 thing, I might lean slightly towards posting multiple tickets since that would require copies of multiple "different" platforms, but it's a really close call. Likewise, the big overhaul posts _could_ be split into multiple tickets, but we have no problem with the omniposts.

(That said, if you split up the tasks into multiple tickets, it makes it more likely you'll get at least some of the changes tackled sooner rather than later; for example, modifying loadouts for multiple aircraft is a massive PITA but swapping sensors isn't so bad, so if that was two tickets I might hit the easier one first and then come around for a second pass on the loadouts ~if~ when I have a freer workday.)

The happy middle ground is to do what @claudejdev has done, e.g. #641; I would consider his posts to be exemplar "omniposts". Breaking big requests into multiple subtasks makes our job much easier, better enables transparency, and also keeps the general ticket count down / minimizes the chance of us missing a related ticket (which happens pretty often when we're using filters for triage).
